src/main.py

```diff
@@ -0,0 +1,131 @@
+import preprocess.etl as etl
+import model.fitmodel as fitmodel
+import forecast.prediction as prediction
+
+import pathlib as pl
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn.naive_bayes import GaussianNB
+from sklearn.svm import SVC
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.ensemble import RandomForestClassifier
+
+if __name__ == "__main__":
+    # step 1: ETL
+    # read in raw data
+    input_data_path = pl.Path(__file__).resolve().parents[1].joinpath('data/input/')  # the parent directory of
+    check_data = etl.Check(input_data_path)
+    df_all_files = check_data.list_all_files()
+    check_data.check_path(df_all_files)
+    check_data.convert(df_all_files)
+    df_raw = check_data.load(df_all_files)
+    check_data.check_size(df_raw)
+
+    # transformation
+    df_transformation = etl.Transformation(df_raw)
+    df_rv_na = df_transformation.remove_missing_value()
+    df_encode = df_transformation.preprocess(df_rv_na)
+
+    # Step 2: Fit models
+    # train test split
+    df_split = fitmodel.AsNumpy(df_encode)
+    train_x, train_y, test_x, test_y, cols = df_split.train_test_split()
+
+    # run a logistic regression model
+    logit = LogisticRegression(C=1.0, class_weight=None, dual=False, fit_intercept=True,
```

Comment:
You defined a model, you didn't "run" it. Your comment above is inaccurate

Reply:
yes and comments updated